.github/workflows/build.yml

```diff
@@ -46,13 +43,13 @@ jobs:
       - run: ./build_oc.tool
 
       - name: Upload to Artifacts
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
           name: macOS XCODE5 Artifacts
           path: Binaries/*.zip
       - name: Upload to Release
         if: github.event_name == 'release'
-        uses: svenstaro/upload-release-action@e74ff71f7d8a4c4745b560a485cc5fdb9b5b999d
+        uses: svenstaro/upload-release-action@v2
```

Comment:
Please use a commit corresponding to their latest tag (release). We cannot entirely trust third-party repositories (non-GitHub) are not compromised and get their tags overwritten.

Reply:
fa9af0fee6b40c5b5399672c697ab032f968225c